objects/ClueCounterSwapper.d919d6/Clues.3f22e5.json

```diff
@@ -33,11 +33,11 @@
   "IgnoreFoW": false,
   "LayoutGroupSortIndex": 0,
   "Locked": true,
-  "LuaScriptState": "[true,0]",
-  "LuaScript_path": "ClueCounterSwapper.d919d6/Resources.3f22e5.ttslua",
+  "LuaScript": "require(\"playermat/GenericCounter\")",
+  "LuaScriptState": "0",
```

Comment:
Is this right?  The LuaScriptState is typically JSON, will this parse correctly?

Reply:
Yeah, this is correct (it's just a single value that is saved)